bin/merge-open

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+"""
+Turn Genbank and RKI metadata & sequences into merged open data
+"""
+
+import typer
+
+
+def main(
+    input_rki_sequences: str = typer.Option(...),
+    input_rki_metadata: str = typer.Option(...),
+    input_genbank_sequences: str = typer.Option(...),
+    input_genbank_biosample: str = typer.Option(...),
+    input_genbank_metadata: str = typer.Option(...),
+    output_sequences: str = typer.Option(...),
+    output_metadata: str = typer.Option(...),
+):
+    import pandas as pd
+    from Bio import SeqIO
+    from xopen import xopen
+
+    # Load metadata files
+    with xopen(input_rki_metadata, "r") as fin:
+        rki = pd.read_csv(fin, index_col="strain", low_memory=False, sep="\t")
+
+    with xopen(input_genbank_metadata, "r") as fin:
+        genbank = pd.read_csv(
+            fin, index_col="strain", low_memory=False, sep="\t"
+        )
+
+    with xopen(input_genbank_biosample, "r") as fin:
+        biosample = pd.read_csv(
+            fin, index_col="biosample_accession", low_memory=False, sep="\t"
+        )
+
+    # Get biosample accessions that are in genbank metadata
+    biosample_accessions = genbank["biosample_accession"].dropna().tolist()
+
+    # Get internal_ids from the biosample accessions that are in genbank metadata
+    internal_ids = biosample[
+        biosample.index.isin(biosample_accessions)
+    ]["internal_id"].dropna()
```

Comment:
We could do that - also thought about this after I went down this road. 

The internal id can be very long, so this would bloat more than adding the biosample accession.

Reply:
The `internal_id` column could just be a temporary column. We can use it here to check for RKI duplicates, then drop the column in the final merged output.

Reply:
Like the biosample_accession column currently - that's true. I'll have a go

Reply:
Done